### PR TITLE
v5 of editor plus fix issue with link management / dialogs

### PR DESCRIPTION
**Pull request by burtonator**
This fixes dialog and link management on mobile. 

Don't worry too much about testing just yet. I still have to do work on mobile but hopefully this will be the last PR and we can start implementing it within the app.  

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/9621

## Description of Changes
- The link dialog won't work on mobile. It works on desktop but we need to replace it. Use our own link dialog.
- Verify the URL handling works on a 320px device.
-  mobile: creating links doesn't work - might need to use our own dialog box. It also causes the keyboard go to away when clicked.
- disable tooltips on mobile
- implement different link handling strategy for mobile and get feedback from @sssssabinaaa on my plan to fix it.
- 
## Test Plan
- 

## Deployment Plan

1. 

## Other Considerations

-

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/components/MarkdownEditor/customLinkDialog/CustomLinkDialogForDesktop.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.CustomLinkDialogForDesktopPopover {
+  max-width: 1000px;
+}
+
+.CustomLinkDialogForDesktop {
+  //min-width: 400px;
+}
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/components/MarkdownEditor/customLinkDialog/CustomLinkDialogForDesktop.scss:7]: remove if not used

**Review by masvelio — APPROVED** [review]

File: packages/commonwealth/client/scripts/views/components/MarkdownEditor/customLinkDialog/CustomLinkEdit.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  if (linkDialogState.type !== 'edit') {
+    return null;
+  }
+
+  return (
+    <div className="CustomLinkEdit">
+      <input
+        type="text"
+        value={link}
+        style={{ flexGrow: 1 }}
```
Comment (mzparacha) [inline packages/commonwealth/client/scripts/views/components/MarkdownEditor/customLinkDialog/CustomLinkEdit.tsx:45]: lets use css

**Review by mzparacha — APPROVED** [review]

File: packages/commonwealth/client/scripts/views/components/MarkdownEditor/customLinkDialog/CustomLinkEdit.tsx (hunk 2 above)
Comment (burtonator) [reply packages/commonwealth/client/scripts/views/components/MarkdownEditor/customLinkDialog/CustomLinkEdit.tsx]: Snuck in! Thanks!

File: packages/commonwealth/client/scripts/views/components/MarkdownEditor/customLinkDialog/CustomLinkDialogForDesktop.scss (hunk 1 above)
Comment (burtonator) [reply packages/commonwealth/client/scripts/views/components/MarkdownEditor/customLinkDialog/CustomLinkDialogForDesktop.scss]: Fixed.

**Pull request merged**